src/model/AbstractTSConfigGenerator.ts

```diff
@@ -6,16 +6,21 @@ import * as path from 'path';
 import * as fs from 'fs';
 import CplacePlugin from './CplacePlugin';
 import { debug } from '../utils';
+import { ExtraTypesReader } from './ExtraTypesReader';
 
 export abstract class AbstractTSConfigGenerator {
     protected tsConfig: any;
-    protected readonly pathToMain: string;
-    protected readonly relPathToPlatform: string;
-    protected readonly relPathToPlatformTs: string;
-    protected readonly platformPlugin = 'cf.cplace.platform';
-    private readonly relRepoRootPrefix = '../../..';
+    protected readonly platformPluginName = 'cf.cplace.platform';
+    protected readonly platformPlugin: CplacePlugin | undefined;
     protected readonly tsConfigJson = 'tsconfig.json';
     protected mainFolderName = '';
+    protected readonly DEST_DIR = 'generated_js';
```

Comment:
From the upper case writing I would expect that this is a static constant. As it is a readonly member I'd suggest
```suggestion
    protected readonly destDir = 'generated_js';
```